drivers/mhz19/include/mhz19_params.h

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Adapt to recent (SAUL) param unification.

Reply:
Should be adapted now